storage/remote/queue_manager.go

```diff
@@ -169,13 +200,17 @@ func newQueueManagerMetrics(r prometheus.Registerer, rn, e string) *queueManager
 		Help:        "The number of shards that the queues shard calculation wants to run based on the rate of samples in vs. samples out.",
 		ConstLabels: constLabels,
 	})
-	m.bytesSent = prometheus.NewCounter(prometheus.CounterOpts{
+
+	bytesSentCounterVec := prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace:   namespace,
 		Subsystem:   subsystem,
 		Name:        "sent_bytes_total",
-		Help:        "The total number of bytes sent by the queue.",
+		Help:        "The total number of bytes sent by the queue after compression.",
 		ConstLabels: constLabels,
-	})
+	}, []string{"type"})
```

Comment:
Type should be avoided as a label name as it's so generic.

Why does this use a label and the others a metric name?

Reply:
refer to https://github.com/prometheus/prometheus/pull/6815#discussion_r392482745

Reply:
That doesn't answer my question and indeed agrees with me. Be consistent in your metrics, it looks like these should all be labels.

Reply:
As I wrote above I disagree that these should all be labels, though pending good use cases I may change my mind. It makes sense to aggregate bytes together across multiple request types to see the total network usage for remote write, so I think a label here makes sense.